content/articles/convert-html-to-pdf-with-docraptor/index.md

```diff
@@ -0,0 +1,205 @@
+               <span class="text-offset" style="color: #807c80;  font-size: 12px">by Michael Feathers</span>
+            </td>
+            <td class="item-stock" style="text-align: center;">In Stock</td>
+            <td class="item-qty" style="text-align: center;">1</td>
+            <td class="item-price" style="text-align: right;">$30.17</td>
+          </tr>
+        </tbody>
+        <tfoot style="text-align: right;">
+          <tr class="text-offset" style="color: #807c80;  font-size: 12px">
+            <td colspan="3"style="padding: 10px 15px; vertical-align: middle;">Subtotal</td>
+            <td>$135.36</td>
+          </tr>
+          <tr class="text-offset" style="color: #807c80;  font-size: 12px">
+            <td colspan="3" style="padding: 10px 15px; vertical-align: middle;">Tax</td>
+            <td>$13.54</td>
+          </tr>
+          <tr>
+            <td colspan="3" style="padding: 10px 15px; vertical-align: middle;">Total</td>
+            <td>$148.90</td>
+          </tr>
+        </tfoot>
+      </table>
+    </div>
+</body>
+</html>");    //supply content directly 
+```
+
+Notice that at times, we may want to convert a document that only exists in the internet space as opposed to having it locally on our machines. Therefore, we need to provide the link to the document we plan to use for such an instance. This process can be done as follows:
+
+```php
+$docraptor = newDocRaptor\DocApi();
+$doc = newDocRaptor\doc();
+$doc = setDocumentUrl("http: //DocRaptor.com/returns/sales.html");  //or use a url
+```
+
+Whether you use HTML or URLs in your report, all of the resources you reference should be accessible via the internet for DocRaptor to work. So we need to have an internet connection for that to take place.
+
+### API document options
+Given that DocRaptor may also convert documents into other files, for instance, HTML to Excel, we must ensure that we create the correct document. Hence, set the document type as PDF.
+
+```php
+$doc->setDocumentType("pdf");   //pdf or xls or xlsx //helps you find the document later
+```
+
+Additionally, we should create a test document because we are using a free API key. When the test is set to `true`, we need to note that the generated document will be watermarked, limited to five downloads, and expire after five days.
+
+```php
+$doc->setTest(true);    //test document are free but watermarked
+```
+
+Another essential thing to note is that, by default, JavaScript is off, but you need to enable it to enhance speed during the document creation process.
+
+```php
+$doc->setJavaScript('true');    //enable JavaScript processing 
+```
+
+### Get the API documentation 
+Following the configuration of your API, run this code to generate your file. Document creation is a one-line code. It also includes an error-handling mechanism that consolidates mistakes in managing the structure if something goes wrong.

```

Comment:
Use a more clear heading, API documentation sounds more like a documentation of the API.